client/src/Profile.js

```diff
@@ -0,0 +1,223 @@
+                className="NameInput"
+                type="text"
+                placeholder="Enter name:"
+                value={name}
+                onChange={e => setName(e.target.value)}
+              ></input>
+
+              <input
+                className="smallInput"
+                type="text"
+                placeholder="Enter location:"
+                value={location}
+                onChange={e => setLocation(e.target.value)}
+              ></input>
+
+              <input
+                className="smallInput"
+                type="text"
+                placeholder="Enter website link:"
+                value={website}
+                onChange={e => setWebsite(e.target.value)}
+              ></input>
+
+              <input
+                className="smallInput"
+                type="text"
+                placeholder="Enter email address:"
+                value={email}
+                onChange={e => setEmail(e.target.value)}
+              ></input>
+
+              <input
+                className="smallInput"
+                type="text"
+                placeholder="Enter phone number:"
+                value={phone}
+                onChange={e => setPhone(e.target.value)}
+              ></input>
+            </div>
+          </div>
+
+          <div className="tagsContainer">
+            {tags &&
+              tags.map((text, index) => (
+                <Tag
+                  key={index}
+                  text={text}
+                  index={index}
+                  edit={edit}
+                  onDelete={handleDelete}
+                />
+              ))}
+          </div>
+        </div>
+        <div className="descriptions">
+          <hr />
+          <div className="header">Mission Statement</div>
+          <textarea
+            className="missionInput"
+            type="text"
+            placeholder="Enter mission statement:"
+            maxLength="350"
+            value={mission}
+            onChange={e => setMission(e.target.value)}
+          ></textarea>
+          <hr />
+          <div className="header">Description</div>
+          <textarea
+            className="descriptionInput"
+            type="text"
+            placeholder="Enter description:"
+            maxLength="1000"
+            value={description}
+            onChange={e => setDescription(e.target.value)}
+          ></textarea>
+          <hr />
+        </div>
+        <div className="socialsDiv">
+          <input
+            className="smallInput"
+            type="text"
+            placeholder="Enter facebok link:"
+            value={fbLink}
+            onChange={e => setFbLink(e.target.value)}
+          ></input>
+          <input
+            className="smallInput"
+            type="text"
+            placeholder="Enter instagram link:"
+            value={instaLink}
+            onChange={e => setInstaLink(e.target.value)}
+          ></input>
+        </div>
+      </div>
+    );
+  }
```

Comment:
this is a really long return statement and hard to read -- try to break it up into individual functions like `renderSocials` or `renderDescriptions` to make it more clear